src/modules/home/Card.jsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { H3, P } from "../../common/components/elements/Text";
-import { Button } from "../../common/components/elements/Button";
+import { LinkedItemStyled } from "../../common/components/elements/LinkedItem";
 
-export const Card = ({ title, img, text, button }) => {
+export const Card = ({ title, img, text, button, link }) => {
```

Comment:
```suggestion
export const Card = ({ title, img, text, button, href }) => {
```

Lets call it href itself in all the pages. make necessary changes